collections/_projects/libremesh-automation_monitoring.md

```diff
@@ -0,0 +1,84 @@
+---
+name: "Automation tools for LibreMesh firmware build and monitoring"
+desc: "Automate the compilation system, configuration and monitoring of LibreMesh using Ansible. Integrate with its web interface Lime-App"
+requirements:
+ - "Analyzing and coding"
+ - "Prometheus"
+ - "Grafana"
+ - "Ansible"
+ - "OpenWrt"
+ - "lua, bash"
+ - "publish code and doc"
+ - "report the process"
+difficulty: "medium"
+size: "350 hours"
+issues:
+ - "https://github.com/coala/coala/issues/####"
+mentors:
```

Comment:
can you please use github usernames for mentors? Links are created based on that.

Reply:
Uh, we didn't know that :)
My username on Github is `ilario`

Reply:
Oh, @digitigrafo already fixed, thanks!

Reply:
yes, mine digitigrafo, I just commit in patch-1